discovery-provider/src/api/v1/helpers.py

```diff
@@ -12,42 +11,30 @@
 from src.queries.get_undisbursed_challenges import UndisbursedChallengeResponse
 from src.queries.query_helpers import LibraryFilterType, SortDirection, SortMethod
 from src.queries.reactions import ReactionResponse
-from src.utils.get_all_other_nodes import get_all_healthy_content_nodes_cached
+from src.utils.auth_middleware import MESSAGE_HEADER, SIGNATURE_HEADER
 from src.utils.helpers import decode_string_id, encode_int_id
-from src.utils.redis_connection import get_redis
-from src.utils.rendezvous import RendezvousHash
 from src.utils.spl_audio import to_wei_string
 
-redis = get_redis()
 logger = logging.getLogger(__name__)
 
 
 def make_image(endpoint, cid, width="", height=""):
     return f"{endpoint}/content/{cid}/{width}x{height}.jpg"
 
 
-def get_primary_endpoint(user, cid):
-    if not cid:
+def get_primary_endpoint(user):
```

Comment:
I don't think we want to revert this

Reply:
omg oops

Reply:
thank you!!

Reply:
me neither, very odd! looks like it was this commit https://github.com/AudiusProject/audius-protocol/pull/5695/commits/edb0552a5c287e2f477fd2770886b6013d36d1d5

So maybe could fix by checking that file out at current main and redoing the edits?
```
git checkout main -- discovery-provider/src/api/v1/helpers.py
```